src/core_plugins/kibana/public/home/components/recently_accessed.js

```diff
@@ -0,0 +1,206 @@
+import './recently_accessed.less';
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  EuiPanel,
+  EuiLink,
+  EuiText,
+  EuiTextColor,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiPopover,
+  EuiIcon,
+  EuiSpacer,
+} from '@elastic/eui';
+
+export const NUM_LONG_LINKS = 5;
+
+export class RecentlyAccessed extends React.Component {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      isPopoverOpen: false,
+    };
+  }
+
+  onButtonClick() {
```

Comment:
can avoid the .bind(this) for this function and closePopover by using the same notation as renderDropdown, e.g.:

```
closePopover = () => {
```